apis/projectcontour/v1alpha1/contourdeployment.go

```diff
@@ -140,6 +140,13 @@ type ContourSettings struct {
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=42
 	DisabledFeatures []contour_v1.Feature `json:"disabledFeatures,omitempty"`
+
+	// CertLifetime is the number of days for which certificates will be valid.
+	// defaults to 365.
+	//
+	// +kubebuilder:validation:Minimum=0
+	// +optional
+	CertLifetime uint32 `json:"certLifetime,omitempty" yaml:"certLifetime,omitempty"`
```

Comment:
It seems we sometimes have `yaml` in the struct tags, but it has been used very inconsistently and mostly we have just `json`. Should we just remove `yaml` here? I assume it is not needed.